doc/user/content/sql/system-catalog.md

```diff
@@ -381,12 +381,12 @@ Field       | Type       | Meaning
 The `mz_peek_active` source describes all read queries ("peeks") that are
 pending in the dataflow layer.
 
-Field    | Type       | Meaning
----------|------------|--------
-`uuid`   | [`text`]   | The ID of the connection that requested the peek. `uuid` is a misnomer; connection IDs are 32-bit unsigned integers.
-`worker` | [`bigint`] | The ID of the worker thread servicing the peek.
-`id`     | [`text`]   | The ID of the index the peek is targeting.
-`time`   | [`bigint`] | The timestamp the peek has requested.
+Field      | Type       | Meaning
+-----------|------------|--------
+`id`       | [`uuid`]   | The ID of the peek request.
+`worker`   | [`bigint`] | The ID of the worker thread servicing the peek.
+`index_id` | [`text`]   | The ID of the index the peek is targeting.
+`time`     | [`bigint`] | The timestamp the peek has requested.
```

Comment:
You'll need to update https://github.com/MaterializeInc/materialize/blob/28d5482e277289349debf5f3c84c74796e099a87/src/dataflow-types/src/logging.rs#L193-L198 to match as well!

Reply:
Done